UniqueList.Tests/UniqueListTests.cs

```diff
@@ -0,0 +1,66 @@
+// <copyright file="UniqueListTests.cs" company="Anna Agafonova">
+// Copyright (c) 2024 Anna Agafonova.
+// Use of this source code is governed by an MIT-style license
+// that can be found in the LICENSE file or at https://opensource.org/licenses/MIT.
+// </copyright>
+namespace UniqueList.Tests;
+using UniqueListLibrary;
+
+public class UniqueListTests
+{
+    private UniqueList<int> uniqueList;
+
+    [SetUp]
+    public void Setup()
+    {
+        uniqueList = new UniqueList<int>();
+    }
+
+    [Test]
+    public void Add_AddExistingElement_ShouldThrowAddExistingElementException()
+    {
+        uniqueList.Add(4);
+
+        Assert.Throws<AddExistingElementException>(() => uniqueList.Add(4));
+    }
+
+    [Test]
+    public void RemoveElement_RemoveNonExistentElement_ShouldThrowDeleteNonExistentElementException()
+    {
+        Assert.Throws<DeleteNonExistentElementException>(() => uniqueList.RemoveElement(4));
+    }
+
+    [Test]
+    public void Add_AddNewNonExistentElement_ShouldSuccessfullyAddElement()
```

Comment:
Add во второй части не нужно повторять, мы там сообщаем особенности теста и его входных данных

А в последнем мы говорим, что должно произойти, то есть элемент должен быть успешно добавлен

```suggestion
    public void Add_NewNonExistentElement_ShouldBeSuccessfullyAdded()
```